toolchain/sem_ir/inst_namer.cpp

```diff
@@ -152,7 +152,15 @@ auto InstNamer::GetNameFor(ScopeId scope_id, InstId inst_id) const
   if (!inst_name) {
     // This should not happen in valid IR.
     std::string str;
-    llvm::raw_string_ostream(str) << "<unexpected instref " << inst_id << ">";
+    llvm::raw_string_ostream str_stream(str);
+    str_stream << "unexpected." << inst_id;
+    auto loc_id = sem_ir_.insts().GetLocId(inst_id);
+    // TODO: Consider handling inst_id cases.
+    if (loc_id.is_node_id()) {
+      auto token = parse_tree_.node_token(loc_id.node_id());
+      str_stream << ".loc" << tokenized_buffer_.GetLineNumber(token) << "_"
+                 << tokenized_buffer_.GetColumnNumber(token);
+    }
```

Comment:
Including the location here seems nice. That said, I would prefer to keep the angle brackets here, both for consistency with other things we print this way, and to make it clearer that this output is not valid SemIR (while we may not have plans to parse SemIR ourselves, other tools might -- it's also nice to make it easier to see in test output when the test produced these bad outputs).

Reply:
Done